### PR TITLE
Removed galaxy bedsheet

### PR DESCRIPTION
**Pull request by John-Dang**
## About The Pull Request
Removes the galaxy bedsheet from naturally spawning in the game.

## Why It's Good For The Game
It doesn't fit in, LRP, and this was requested by one of the heads.

## Changelog
:cl:
del: Removed cosmic space bedsheet from the map and the table of bedsheets that spawn from random sheet spawners.
tweak: Slightly tweaked a poster description.
/:cl:

### CODE REVIEW

**Comment by thedarkcrusader** [comment]
whats uh LRP about it

**Comment by thedarkcrusader** [comment]
Fact lets sit here for a moment how the fuck do you low rp with a bedsheet?

**Comment by Rohesie** [comment]
Remove the sprite as well then.

**Comment by cornercube** [comment]
what is the exact problem with a space-themed bedsheet?
it is very rarely used to meme on anyone

**Comment by John-Dang** [comment]
> Fact lets sit here for a moment how the fuck do you low rp with a bedsheet?

Its the word from on high and I'm just taking my marching orders, and brother am I marching to them.

**Comment by John-Dang** [comment]
> what is the exact problem with a space-themed bedsheet?
> it is very rarely used to meme on anyone

Bossman says to remove the meme sheet, and I remove the meme sheet. If I disagreed I wouldn't be working on this.

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -89,7 +89,7 @@ GLOBAL_LIST_INIT(dye_registry, list(
 		DYE_CE = /obj/item/bedsheet/ce,
 		DYE_RD = /obj/item/bedsheet/rd,
 		DYE_CMO = /obj/item/bedsheet/cmo,
-		DYE_COSMIC = /obj/item/bedsheet/cosmos
+		DYE_COSMIC = /obj/item/bedsheet/nanotrasen
```
Comment (Rohesie) [inline code/game/machinery/washing_machine.dm:92]: remove `DYE_COSMIC` as well

**Comment by Rohesie** [comment]
The icons for `/obj/item/clothing/suit/hooded/wintercoat/cosmic` and `/obj/item/clothing/head/hooded/winterhood/cosmic` need to go as well.

**Pull request merged**